models/tasks.js

```diff
@@ -226,9 +243,24 @@ const getNewTask = async (skill, level) => {
  * @returns {Promise<task>|object}
  */
 
-const fetchSkillLevelTask = async (skill, level) => {
+const fetchSkillLevelTask = async (userId) => {
   try {
-    const task = await getNewTask(skill, level);
+    let task;
+    const data = await ItemModel.where("itemId", "==", userId).where("tagType", "==", "SKILL").limit(10).get();
```

Comment:
Won't this always return the first 10 data only, how will we get the next 10?

Reply:
mostly 10 will do the work for us, we do not want all the skills of the user

Reply:
Won't this return the same skill each and every time?